login-workflow/src/components/BrandedCardContainer/BrandedCardContainer.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { ReactNode } from 'react';
+import Card, { CardProps as CardPropsType } from '@mui/material/Card';
+import { SxProps, Theme, useTheme } from '@mui/material/styles';
+import { Spinner, SpinnerProps } from '../Spinner';
+import defaultBackgroundImage from '../../assets/images/background.svg';
+import { useInjectedUIContext } from '@brightlayer-ui/react-auth-shared';
+import Box, { BoxProps } from '@mui/material/Box';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+import {
+    getBrandedCardContainerUtilityClass,
+    BrandedCardContainerClassKey,
+    BrandedCardContainerClasses,
+} from './BrandedCardContainerClasses';
+import { cx } from '@emotion/css';
+
+export type BrandedCardContainerProps = BoxProps & {
+    loading?: boolean;
+    backgroundImage?: string;
+    cardStyles?: SxProps<Theme>;
+    CardProps?: CardPropsType;
+    LoaderComponent?: ReactNode;
```

Comment:
This should be `loaderComponent` unless there's a naming convention I'm missing

Reply:
yes, a naming convention